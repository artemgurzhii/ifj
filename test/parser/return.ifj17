
# return without a value
def foo()
	return
end

# return with value
def bar(): string
	return "foo"
end