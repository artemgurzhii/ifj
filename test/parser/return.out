(function foo ->
  (return))

(function bar -> (id string)
  (return
    (string 'foo')))